src/utils/components/ListPageFilter/hooks.ts

```diff
@@ -0,0 +1,87 @@
+import { useCallback, useMemo } from 'react';
+import { useHistory, useLocation } from 'react-router-dom';
+
+type UseRowFiltersParametersType = (inputs: {
+  filterKeys: { [key: string]: string };
+  filters: { [key: string]: string[] };
+}) => [string[], (selected: string[]) => void];
+
+export const useRowFiltersParameters: UseRowFiltersParametersType = ({ filterKeys, filters }) => {
+  const location = useLocation();
+  const { setOrRemoveQueryArgument } = useQueryParamsMethods();
+  const queryParams = useMemo(() => new URLSearchParams(location.search), [location]);
+
+  const selectedRowFilters = useMemo(
+    () =>
+      Object.values(filterKeys)
```

Comment:
u think better add .? of Object.XX failes we are getting a crash

Reply:
for map & flat

Reply:
i'll add `|| {}` so `selectedRowFilters` is always defined